pyproject.toml

```
@@ -51,19 +53,52 @@ dependencies = [
   "tomlkit>=0.10.1",
   "typing-extensions>=3.10; python_version<'3.10'",
 ]
+
+# Extras are optional dependencies that can be installed with the package
 optional-dependencies.spelling = [ "pyenchant~=3.2" ]
 optional-dependencies.testutils = [ "gitpython>3" ]
-urls."Bug Tracker" = "https://github.com/pylint-dev/pylint/issues"
-urls."Discord Server" = "https://discord.com/invite/Egy6P8AMB5"
-urls."Docs: Contributor Guide" = "https://pylint.readthedocs.io/en/latest/development_guide/contributor_guide/index.html"
-urls."Docs: User Guide" = "https://pylint.readthedocs.io/en/latest/"
-urls."homepage" = "https://github.com/pylint-dev/pylint"
-urls."Source Code" = "https://github.com/pylint-dev/pylint"
-urls."What's New" = "https://pylint.readthedocs.io/en/latest/whatsnew/3/"
-scripts.pylint = "pylint:run_pylint"
-scripts.pylint-config = "pylint:_run_pylint_config"
-scripts.pyreverse = "pylint:run_pyreverse"
-scripts.symilar = "pylint:run_symilar"
+
+[dependency-groups]
+# Dependency groups store dependencies that should not be distributed with the package
+dev = [
+  "contributors-txt>=1",
+  "pre-commit",
+  "tbump~=6.11.0",
+]
+
+test = [
+  "coverage~=7.8",
+  "pytest-cov~=6.0",
+  "pytest-xdist~=3.6",
+  "six",
+  "tox>=3",
+  "types-setuptools==78.1.0.20250329",
+  { include-group = "test-min" },      # Include test-min dependencies
```

```suggestion
  { include-group = "test-min" },
```

This seems a bit redundant